### PR TITLE
refactor: update wallet screens - onboarding

### PR DESCRIPTION
**Pull request by Jennievon**
# :books: Linear Ticket :books:

[#LIQ-699](https://linear.app/liquality/issue/LIQ-699/integrate-tandc-and-success-screen-in-import-wallet-flow-missing)
[#LIQ-703](https://linear.app/liquality/issue/LIQ-703/align-pw-creation-step-in-wallet-creation-with-import-flow-and)

### CODE REVIEW

File: src/views/Splash.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
             : { path: '/onboarding/home', query: { isImport: true } }
         "
         ><p class="text-center" id="import_with_seed_phrase_option">
-          Import with seed phrase
+          Forgot password? Import with seed phrase
```
Comment (monokh) [inline src/views/Splash.vue:17]: If I just installed the wallet. It still say's "Forgot password?" But it shouldn't

File: src/views/UnlockWallet.vue [hunk 2]
```diff
@@ -66,6 +66,7 @@ export default {
       password: ''
     }
   },
+  computed: mapState(['termsAcceptedAt']),
```
Comment (monokh) [inline src/views/UnlockWallet.vue:69]: You added this but its not used?

**Review by monokh — CHANGES_REQUESTED** [review]

File: src/views/Splash.vue (hunk 1 above)
Comment (Jennievon) [reply src/views/Splash.vue:17]: I had that implemented but the design shows this:

Screenshot 2022-01-18 at 15 43 21

File: src/views/UnlockWallet.vue (hunk 2 above)
Comment (Jennievon) [reply src/views/UnlockWallet.vue:69]: Taking it out now.

File: src/views/Splash.vue (hunk 1 above)
Comment (monokh) [reply src/views/Splash.vue]: i mean that's surely wrong? 😄  There is no password at that stage.

Comment (Jennievon) [reply src/views/Splash.vue]: I agree. I have changed it to just "Import with seed phrase" now.

**Review by monokh — APPROVED** [review]

**Pull request merged**